### PR TITLE
chore: Uses only latest provider version for Test Suite workflow

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Uses only latest provider version for Test Suite workflow.

Link to any related issue(s): https://jira.mongodb.org/browse/CLOUDP-218767

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [X] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [X] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [X] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [X] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [X] I have added any necessary documentation (if appropriate)
- [X] I have run make fmt and formatted my code
- [X] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -39,7 +39,10 @@ We use a tool called [github changelog generator](https://github.com/github-chan
     docker run -it --rm -v "$(pwd)":/usr/local/src/your-app githubchangeloggenerator/github-changelog-generator -u mongodb -p terraform-provider-mongodbatlas -t <GH_TOKEN> --breaking-labels "breaking-change" --enhancement-label "**Enhancements**" --bugs-label "**Bug Fixes**"  --issues-label "**Closed Issues**" --pr-label "**Internal Improvements**" --max-issues 1000
     ```
     To obtain your github personal access token you can use the following guide: [Authorizing a personal access token for use with SAML single sign-on](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on)
- 
+
+#### Update Test Suite versions
+- Update [Test Suite workflow](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github/workflows/test-suite.yml) if needed. Check `provider_matrix` and `schedule_provider_matrix` to make sure that they include the latest patch of the latest 2 provider versions.
```
Comment (andreaangiolillo) [inline RELEASING.md]: I am not sure what the latest patch of the latest 2 provider versions means 🤔

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: RELEASING.md (hunk 1 above)
Comment (lantoli) [reply RELEASING.md]: you're right it's confusing. what about this?  "... the latest patch version release of the latest 2 major versions"

at this moment, the 2 latest versions of the provider are 1.14.x and 1.13.x, and the latest patches make: 1.14.0, 1.13.1

do you have any suggestion?

Comment (andreaangiolillo) [reply RELEASING.md]: Maybe you can also provide an example

Comment (lantoli) [reply RELEASING.md]: finally we decided to use only the latest one at the moment

**Pull request merged**